hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.0__create_tables.sql

```diff
@@ -116,6 +116,7 @@ comment on table contract is 'Contract entity';
 create table if not exists contract_action
 (
     call_depth          integer                        not null,
+    call_operation_type smallint                       null,
```

Comment:
There should always be an operation type. Add from services side at worst it'll be unknown iirc
```suggestion
    call_operation_type smallint                       not null,
```

Reply:
will this work for existing rows?

Reply:
you'll have to first set a default value and not null in the create, then drop the default value. clarification: only needed for V1 migration files. V2 always starts with an empty db or good data.

Reply:
There are no existing rows at this time, so it's safe.